plugin-viewport/src/main/kotlin/com/mapbox/maps/plugin/viewport/experimental/transition/TransitionUtils.kt

```diff
@@ -0,0 +1,67 @@
+package com.mapbox.maps.plugin.viewport.experimental.transition
+
+import android.animation.Animator
+import android.animation.AnimatorSet
+import com.mapbox.geojson.Point
+import com.mapbox.maps.plugin.delegates.MapCameraManagerDelegate
+import kotlin.math.hypot
+import kotlin.math.pow
+import kotlin.math.roundToInt
+
+internal fun createAnimatorSet(animators: List<Animator>) = AnimatorSet().apply {
+  playTogether(*animators.toTypedArray())
+}
+
+internal fun screenDistanceFromMapCenterToTarget(
+  cameraManager: MapCameraManagerDelegate,
+  currentCenter: Point,
+  targetCenter: Point
+): Double {
+  val currentCenterScreenCoordinate = cameraManager.pixelForCoordinate(currentCenter)
+  val locationScreenCoordinate = cameraManager.pixelForCoordinate(targetCenter)
+  return hypot(
+    currentCenterScreenCoordinate.x - locationScreenCoordinate.x,
+    currentCenterScreenCoordinate.y - locationScreenCoordinate.y
+  )
+}
+
+/**
+ * Returns a bearing change using the shortest path.
+ */
+internal fun normalizeBearing(currentBearing: Double, targetBearing: Double): Double {
```

Comment:
as I guess I've already stated perhaps in another PR - we already use such utils methods for location component, would be nice to re-use them for shortest bearing path and normalization

Reply:
sure, will take a deeper look in next steps, as this PR mainly validates the public interface.